ktcacheorbust.cc

```diff
@@ -38,28 +38,43 @@ void CacheOrBust::configure(kt::TimedDB* dbary, size_t dbnum,
   std::vector<std::string>::iterator it = elems.begin();
   std::vector<std::string>::iterator itend = elems.end();
   while (it != itend) {
-    std::vector<std::string> fields;
-    if (kc::strsplit(*it, '=', &fields) > 1) {
```

Comment:
any particular reason for the change from this to `std::` strings?

not objecting, just curious what the impact is

Reply:
The original code split the string into a vector on `=` which was fine if there was no ™ in the value. While fixing that, I switched to std:: strings because they had nice operations for finding the = and grabbing the substring. I figured when in C++....